src/Core/src/Fonts/FontManager.Windows.cs

```diff
@@ -146,16 +148,20 @@ IEnumerable<string> GetAllFontPossibilities(string fontFamily)
 			{
 				var fontUri = new Uri(fontFile, UriKind.RelativeOrAbsolute);
 
-				// CanvasFontSet only supports ms-appx:// and ms-appdata:// font URIs
-				if (fontUri.IsAbsoluteUri && (fontUri.Scheme == "ms-appx" || fontUri.Scheme == "ms-appdata"))
+				// unpackaged apps can't load files using packaged schemes
+				if (!AppInfoUtils.IsPackagedApp)
 				{
-					using (var fontSet = new CanvasFontSet(fontUri))
+					var path = fontUri.AbsolutePath.TrimStart('/');
+					if (FileSystemUtils.TryGetAppPackageFileUri(path, out var uri))
+						fontUri = new Uri(uri, UriKind.RelativeOrAbsolute);
```

Comment:
This is the real change for unpackaged, we determine the full path to the font and use that. Packaged still uses the same `ms-appx://`. Technically the full path also works in packaged, but no need to change what works.